export/src/main/scala/org/clulab/reach/export/TrainingDataExporter.scala

```diff
@@ -0,0 +1,88 @@
+package org.clulab.reach.`export`
+
+import org.clulab.odin.{CrossSentenceMention, EventMention, Mention, RelationMention, TextBoundMention}
+import org.json4s.JsonAST.JArray
+import org.json4s._
+import org.json4s.JsonDSL._
+import org.json4s.jackson.JsonMethods._
+
+import scala.annotation.tailrec
+
+case class Datum(
+                words: Seq[String],
+                eventIndices: Range,
+                type_ : String,
+                polarity: Boolean,
+                controllerIndices: Range,
+                controlledIndices: Range,
+                triggerIndices:Range
+                ) {
+  val json: JValue =
+    ("sentence_tokens" -> words.toList) ~
+      ("event_indices" -> eventIndices.toList) ~
```

Comment:
This toList was probably done on purpose.  The alternative is to just record the bounds.

Reply:
Yes, this is for convenience on the client side